segmentation/src/supervoxel_clustering.cpp

```diff
@@ -161,14 +168,14 @@ typedef pcl::octree::OctreePointCloudAdjacencyContainer<pcl::PointXYZ, VoxelData
 typedef pcl::octree::OctreePointCloudAdjacencyContainer<pcl::PointXYZRGB, VoxelDataRGBT> AdjacencyContainerRGBT;
 typedef pcl::octree::OctreePointCloudAdjacencyContainer<pcl::PointXYZRGBA, VoxelDataRGBAT> AdjacencyContainerRGBAT;
 
-template class pcl::SupervoxelClustering<pcl::PointXYZ>;
-template class pcl::SupervoxelClustering<pcl::PointXYZRGB>;
-template class pcl::SupervoxelClustering<pcl::PointXYZRGBA>;
+template class PCL_EXPORTS pcl::SupervoxelClustering<pcl::PointXYZ>;
+template class PCL_EXPORTS pcl::SupervoxelClustering<pcl::PointXYZRGB>;
+template class PCL_EXPORTS pcl::SupervoxelClustering<pcl::PointXYZRGBA>;
```

Comment:
AFAIR `PCL_EXPORTS` is mostly relevant for Windows. Every time you forget to add it to class declaration Windows users come and report unresolved symbols :D. But in this case `PCL_EXPORTS` is already present in the [declaration](https://github.com/stefanbuettner/pcl/blob/633343bb66e4523f582719d4bd6ec219401adf57/segmentation/include/pcl/segmentation/supervoxel_clustering.h#L123) of `SupervoxelClustering`, so indeed I don't see why we need it here.